SofaKernel/framework/sofa/core/visual/DisplayFlags.cpp

```diff
@@ -197,9 +195,7 @@ void FlagTreeItem::read_recursive(FlagTreeItem *root, const std::map<std::string
                         msg_warning("DisplayFlags") << "FlagTreeItem '" << (*iter)->m_showName[i] << "' is deprecated, please use '"<<(*iter)->m_showName[0]<<"' instead";
                     }
 
-                    tristate merge_showhide;
-                    if (show) merge_showhide = tristate::true_value;
-                    (*iter)->setValue(merge_showhide);
+                    (*iter)->setValue(tristate::true_value);
```

Comment:
with previous change, shouldn't be the code be:
```
if (hide) 
    (*iter)->setValue(tristate::false_value);
else if(show)
    (*iter)->setValue(tristate::true_value);
else
    ???
```

Reply:
From a quick view show and hide seem to be 2 different things from 2 different Datas. It seems obvious that it could be unified but it may need deep changes. Needs deeper checking.